elements/pfe-jump-links/demo/index.html

```diff
@@ -57,36 +56,40 @@ <h1><code>pfe-jump-links</code></h1>
       position:sticky;
       top: 0;
     }
+    body {
+      font-family: 'Red Hat Text', sans-serif;
+      font-weight: 400;
+    }
   </style>
       
   <h2 style="text-align: center; margin: 60px auto;">PFE Jump Links - standard</h2>
   <div style="margin-bottom: 400px;" class="pfe-l-grid">
     <div class="sticky pfe-l-grid__item pfe-m-12-col pfe-m-3-col-on-lg">
       <pfe-jump-links-nav id="jumplinks1" default>
-        <h4 class="heading" slot="heading">Jump to section</h4>
+        <h4 class="pfe-jump-links-nav--heading" slot="pfe-jump-links-nav--heading">Jump to section</h4>
         <ul class="pfe-jump-links-nav">
-          <li>
-            <a class="pfe-jump-links-nav__item" href="#section1" data-target="section1">Section 1</a>
+          <li class="pfe-jump-links-nav__item">
+            <a class="pfe-jump-links-nav__link" href="#section1" data-target="section1">Section 1</a>
```

Comment:
```suggestion
            <a class="pfe-jump-links-nav__item--link" href="#section1" data-target="section1">Section 1</a>
```

Since the links are nested children of the item, I think the relationship should be noted by the class name.  By using the underscore to indicate region and the `--link` to indicate a subsection, we can create a clearer relationship between them in our styles.